pkg/detect/ospkg/ospkg.go

```diff
@@ -129,28 +132,29 @@ func Detect(dbc db.DB, sr scanTypes.ScanResult) (detectTypes.VulnerabilityDataDe
 		}
 	}
 
-	contents := make(map[string]map[sourceTypes.SourceID]criteriaTypes.FilteredCriteria)
+	contents := make(map[string]map[sourceTypes.SourceID]conditionTypes.FilteredCondition)
 	for rootID, m := range cm {
-		for sourceID, fca := range m {
-			qs := make([]criterionTypes.Query, 0, len(fca.indexes))
-			for _, idx := range fca.indexes {
+		for sourceID, fs := range m {
```

Comment:
It is better to use either `fs` or `containedList`
https://github.com/MaineK00n/vuls2/pull/84/files#diff-bf1f3d3fc4dd9c7172fda625d2f706506db8624e551ebb6e0adab81affe5dd34R83

Reply:
Nice catch!
https://github.com/MaineK00n/vuls2/compare/be79505f64c1bf5aa71a1182d2a8e0509eceea5a..4493ac52c8e78553815338260e823fa5b18ef277